classes/install/form/InstallForm.inc.php

```diff
@@ -56,7 +56,7 @@ public function __construct($request)
     {
         parent::__construct($request, 'install/install.tpl');
 
-        $this->supportedLocales = array_map(fn(LocaleMetadata $locale) => $locale->getDisplayName(), Locale::getLocales());
+        $this->supportedLocales = array_map(fn(LocaleMetadata $locale) => $locale->getDisplayName(null, true), Locale::getLocales());
```

Comment:
As we're using PHP 8, another possibility is to write: `$locale->getDisplayName(withCountry: true)`, in case you feel it's a cleaner way to skip the optional parameters.

Reply:
It's a good feature to improve the usability of those functions with an infinite list of optional parameters, in this case, it doesn't help much =]

Reply:
Yeah, this feature opens up space for this approach. In general I think it's better to restructure into new classes/extra methods.

Following the same idea, you could create a new method: `getDisplayNameWithCountry()`... It's easier to read/understand and more difficult to break external code (e.g. if you replace `(bool $withCountry = false)` later by `(bool $withScript = false, $withCountry = false)` it will break the user code and nobody will notice).

The tough problem of having too many options to choose from 😅